containers/Dash/Users/index.js

```diff
@@ -39,6 +39,10 @@ class Users extends React.Component {
 
   render() {
     const { users } = this.props
+
+    // Sort by username
+    users.sort(function(a,b) {return (a.username > b.username) ? 1 : ((b.username > a.username) ? -1 : 0);} );
+
```

Comment:
@olalonde good catch. I was noticing the bug after I pushed and finally got around to addressing the issue.

I would love to add the sort options, but I do not have time now. Sorry 😢